src/ir/subtypes.h

```diff
@@ -51,6 +54,13 @@ struct SubTypes {
     return ret;
   }
 
+  // Like getAllSubTypes, but also includes the type itself.
+  std::vector<HeapType> getAllSubTypesInclusive(HeapType type) {
```

Comment:
How about we name this `getAllSubtypes` and rename the current `getAllSubTypes` to `getStrictSubtypes`?

(Changing to a lowercase `t` in `Subtype` is intentional, but also we can leave that to be a global renaming in a future PR)

Reply:
Ok, renamed to `getStrictSubTypes`.

(About removing the capital `T`, I'm not opposed to that, but is it the right spelling? Is "subtype" a single world in English?)

Reply:
Yes, "subtype" is a single word.

Reply:
Is "supertype" also a single word?

Looks like we consider them both as type words atm in the codebase. But sgtm to fix to whatever is the common usage.

Reply:
Yes, supertype is also a single word.